---
layout: page
title: Resume
permalink: /resume/
icon: far fa-address-card
type: page
---

* content
{:toc}


Liangke (Connie) Gou
============

<hr />
<div style="width: 800px;margin: 0 auto;padding: 0;overflow: auto;">
	<div style="float:left;">         
		<p>Department of Human Genetics, UCLA</p>
		<p>Los Angeles, CA90095</p>
		<p>USA</p>
	</div>
	<div style="float:right;">
		<p>lgou@ucla.edu</p>
		<p>gouliangke.github.io</p>
		<p>[LinkedIn](https://www.linkedin.com/in/liangke-gou-9869907a/)</p>
	</div>
</div>

Education
---------

2014-2019 
:   **Doctor of Philosophy, Human Genetics**; University of California, Los Angeles
    
    *GPA 3.8*
     
 2014-2019 
:    **Master of Science, Statistics**; University of California, Los Angeles
    *GPA 3.8*

2010-2014
:   **Master of Science, Biological Science**; Sichuan University
    *GPA 3.9*

Skills
----------------------------------------

* Programming Languages:
     * R
     * Python
     * Julia
     * Keras
     * SQL
     * Unix/Linux
     * bash
     * Git
     * Html
     * Jeckyll

* Statistics
     * Regularized regression
     * Dimension reduction
     * Sampling
     * Bayesian statistics 
     * Missing data
     * Hierarchical models
     * Model selection
     * Latent variables
     * Causal inference
     * Multivariate analysis
     
* Deep/Machine Learning
     * Classification (CNN, adaboost/xgboost, SVM,
      random forests)
     * Unsupervised Learning (clustering, NMF, factor analysis)}     
 
* Biology Techniques
	 * Genetic Engineering using CRISPR-Cas9 system in Stem Cell and Yeast
     * Genetic Cloning
     * Next Generation Sequencing Library Preparation
     * Molecular Cloning
     * DNA/RNA Amplification
     * Stem Cell Culture 
     * FACS Cell Sorting

* Languages:
     * Mandarin (native speaker)
     * English (fluent)


Summary
----------
Ph.D. student with thorough hands-on experience in quantitative genetics and statistics, including mapping the quantitative trait loci for different traits, genetic engineering using CRISPR in yeast and stem cells, analyzing DNA sequencing data, RNA sequencing data and methylation data. Supportive and enthusiastic team player dedicated to streamlining processes and efficiently resolving project issues.


Experience
----------

* Jul. 2015 - Present     Laboratory of Quantitative Genetics, UCLA  
  Graduate Researcher; Advisor Leonid Kruglyak
  
	* Development of High-throughput Assay
	
      Developed and implemented an automated high-throughput fluctuation analysis to measure mutation rate of 1040 yeast segregants. This automated system enabled large scale measure of mutation rate the first time.
    * Quantitative Genetic Analysis
    
      Performed quantitative trait loci (QTL) linkage mapping of 1040 yeast segregants at 28220 genetic markers for the mutation rate variation.
    * Convolutional Neural Network
    
      Predicted the genetic engineering efficiency of different guide RNAs using Convolutional neural network.
    * Genetic Engineering in Yeast and Stem Cells
    
      Modified the targeting genome of yeast and stem cells using the cutting-edge CRISPR-Cas9 system.
      

* Apr. 2015 - Jun. 2015     Laboratory of Computational Population Genetics, UCLA
  Rotation Graduate Student; Advisor Bogdan Pasaniuc
    
    * Causal Inference
          
      Analyze the 1000 genome data and Geuvadis RNA sequencing data and predict causal SNPs in risk loci using a probabilistic framework that integrates association strength with genomic functional annotation data to improve accuracy in selecting plausible causal variants.

* Oct. 2014 - Dec. 2014     Laboratory of Lipid Metabolism, UCLA
  Rotation Graduate Student; Advisor Karen Reue
  
    * Differential Gene Expression Analysis
    
      Investigated the differential expression pattern of micro RNAs in the adipose tissue of two group of mice to test the micro RNA roles in obesity.

* Jul. 2013 - Sep. 2013     Laboratory of Computational Biology, UCLA
  CSST Summer Undergraduate Researcher; Advisor Matteo Pellegrini
  
    * Methylation Data Analysis
    
      Analyzed the high-throughput Reduced Representation Bisulfite Sequencing (RRBS) data.
    * Statistical Modeling         
          
      Examined the association between the methylation level of oral bacteria and the host sexual preference using generalized linear model and hierarchical clustering.
  
     
Publications and Pre-prints
----------
* 2018
  Liangke Gou, Joshua Bloom, Leonid Kruglyak. The genetic basis of mutation rate variation in yeast. bioRxiv
  [https://www.biorxiv.org/content/early/2018/06/04/338723](https://www.biorxiv.org/content/early/2018/06/04/338723)
      
* 2013
  Li-Chun Jiang, Liang-Ke Gou, Xin Zhang, Qing-Mei Zhao, Shuai Tan, Rui Peng, Yu-Qing, Wei\*, \& Fang-Dong Zou\*;Complete mitochondrial genome of a new subspecies of the blue sheep, Pseudois nayaur (Cetartiodactyla: Caprinae) from Helan Mountain in China; Mitochondrial DNA, DOI: 10.3109/19401736.2013.855759
